doc/api/next_api_changes/behavior/18988-TH.rst

```diff
@@ -0,0 +1,5 @@
+The Artist property *rasterized* cannot be *None* anymore
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is now a bool only. Before the default was non and `.Artist.set_rasterized`
```

Comment:
```suggestion
It is now a bool only. Before the default was *None* and `.Artist.set_rasterized`
```

Reply:
We only use *None* as a default, if that default is not a simple data type (e.g. depends on `rcParams` or is an object; e.g. a Norm`. If the default is a bool like here, we use that directly.

Reply:
Did you merge with the typo?

Reply:
Woops. I was so distracted by your confusion :smiley:. Corrected in #18996.